python/tank/util/defaults_manager.py

```diff
@@ -24,6 +24,6 @@ def get_http_proxy(self):
     def get_user(self):
         from . import shotgun
         data = shotgun.get_associated_sg_config_data()
-        if "api_script" in data:
+        if "api_script" in data and data["api_script"]:
```

Comment:
Yeah, I forgot api_script could be set to null. Wouldn't data.get("api_script") be better at that point?